### PR TITLE
development

### PR DESCRIPTION
**Pull request by lucianosarno**
First for linters checking, further for code review.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+}
+
+/* .body__header {
+}
+
+.body__latestNews {
+}
+
+.body__topArticles {
+} */
```
Comment (andresporras3423) [inline css/style.css]: Remove commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+.header {
+  align-content: center;
+  background-color: #ddd;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+/* .header__logo {
+} */
```
Comment (andresporras3423) [inline css/style.css]: Remove commented code

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+  margin: 2px;
+  float: left;
+  position: relative;
+}
+
+.topArticles__topArticleNext {
+  margin: 2px;
+  float: left;
+  position: relative;
+}
```
Comment (andresporras3423) [inline css/style.css]: Group selectors with the same style definition, or create a new class.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+.topicsList::after,
+.topicsList::before {
+  content: "";
+  clear: both;
+  display: block;
+}
+
+.topMenu {
+  width: 100%;
+}
```
Comment (andresporras3423) [inline css/style.css]: Group this selector with the selector of line 386, or create a new class.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,552 @@
...
+    width: 90%;
+  }
+
+  .latestNewImg {
+    width: 90%;
+  }
+
+  .latestNewTitle {
+    width: 90%;
+  }
```
Comment (andresporras3423) [inline css/style.css]: Group selectors with the same style definition, or create a new class.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @lucianosarno @karthy1998** 
Nice job with your first submission for this project :muscle:  
You have made good progress  :clap: :clap: :clap:
Still, your project needs some improvements:
- [ ] the screenshot of your project (in the README file) is not loading
- [ ] Your live demo link is broken, to create a live demo you can use the next link: https://raw.githack.com/
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.
- [ ] remove commented code in your CSS class

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by karthykarthick** [comment]
we use BEM approach

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO @lucianosarno AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT. :computer: 

### GREAT WORK SO FAR :clap: 

**YOU HAVE DONE SUCH A GREAT JOB BUT STILL THERE ARE AFEW CHANGES TO BE MADE** :arrow_down: 

Kindly fix the following: 

- [x] Your project doesn't have a logo. This should be added at the desktop version of your site :smile: 
### YOURS
logo
### ORIGINAL
logo original

- [x] You don't have a latest deal section in your site. Kindly add it

- [ ] Your footer doesn't look like the original kindly you need to fix that
### YOURS
footer section
### ORIGINAL
footer section original

- [ ] Your news section doesn't look like the original kindly make it look like the original site
### YOURS
news section
### ORIGINAL
news section original

- [ ] There is a responsiveness issue with your images at the mobile view. Kindly you need to make sure that your images are not been hidden rather they respond according to the screen size. This should be the behaviour of a good responsive site
responsiveness issue

- [ ] There are some validation errors in your HTML and CSS document kindly fix that
css errors
validation error

**once you have fixed all the above submit your project for another review** :smile: 

### HAPPY CODING :computer:

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
## Status: Changes Required ♻️ 
****
### Hi @karthykarthick and @lucianosarno! 👋 👋 
You guys have done a great job working on the TSE feedback so far!👍Your project looks really good! 👏 👏   There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

### Navigation Bar

- **Medium version**

- [x] The TNW logo should appear on the medium screen size. Please compare your clone with the original page and add this. 
Your clone 👇 
Screen Shot 2020-07-08 at 7 00 57 PM
The original page 👇 
Screen Shot 2020-07-08 at 7 01 19 PM


### Top 3 articles section

- **Large version** 

- [x] The TNW logo should be present in the top left corner of the large image. This was pointed out to you in a previous review. Please add this. 
Your clone 👇 
Screen Shot 2020-07-08 at 7 02 47 PM
The original page 👇 

Screen Shot 2020-07-08 at 7 03 11 PM



- **Medium & Small version**

- [x] The two smaller images on the side do not appear to be responsive. They are being cut off when the screen shrinks. Please fix this in both the small and the medium screen. 
Your clone (medium) 👇 
Screen Shot 2020-07-08 at 7 05 50 PM
Your clone (small) 👇 
Screen Shot 2020-07-08 at 7 06 18 PM
_hint: You can try using `background-size: cover` to fix this_


### Latest news section

- **Large version** 

- [x] This section does not look like the original as was pointed out to you in a previous review. It appears to be left-aligned as there is more space on the right. It should be centered. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-07-08 at 7 11 39 PM
The original page 👇 
Screen Shot 2020-07-08 at 7 12 38 PM
- **Small version**

- [x] It also does not look like the original on the small screen size. The article title as well as the small font should be right next to the image. Please compare your clone with the original and fix this. 
Your clone 👇 
Screen Shot 2020-07-08 at 7 15 21 PM
The original page 👇 
Screen Shot 2020-07-08 at 7 15 40 PM

### Categories (Apps, Gear, Tech, etc) section

- **Large version** 

- [x] You are missing a row of categories in this section. There should be three rows of three categories whereas in yours there is only one row. 

- **Medium version**

- [x] The categories should align to two categories per row in the medium screen. 
Your clone 👇 
Screen Shot 2020-07-08 at 7 19 49 PM
The original page 👇 
Screen Shot 2020-07-08 at 7 20 18 PM

### Latest deals section

- **Small version**

- [x] The same issue from the latest news section is also present in this section. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-07-08 at 7 21 49 PM
The original page 👇 
Screen Shot 2020-07-08 at 7 22 07 PM


### Footer

- [x] Your footer still does not look like the original. This was pointed out to you in a previous review. Please compare your clone with the original page paying close attention to the colors. 
The original page 👇 
Screen Shot 2020-07-08 at 7 23 34 PM
Your clone 👇 
Screen Shot 2020-07-08 at 7 23 49 PM

- [x] Additionally, there is unnecessary white space below your footer. Please ensure that your footer extends all the way to the bottom of the page. 
Screen Shot 2020-07-08 at 7 24 53 PM


### [OPTIONAL]

- Your clone appears to have two "Latest News" sections. Feel free to remove one of these as it is not needed. The additional section is right above the "Latest Deals" Section. 

- This project should not have any media queries for the mobile view as it is a media first project. I am sorry that this was not pointed out to you in previous reviews therefore I will not mark it as a required change. However, you are free to implement this if you would like.
Screenshot of requirements from pathwright 👇 
Screen Shot 2020-07-08 at 7 29 03 PM

- The HTML validator is showing a warning. However, as it is not an error you are not required to fix this.
Screen Shot 2020-07-08 at 7 29 56 PM






****

### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by expjazz — CHANGES_REQUESTED** [review]
## Hey @lucianosarno  and @karthykarthick 
### [status: Change Required]

#### You did a good work on this project, but we still need to changes a few things as pointed out by the TSE's above:

### Top 3 articles section
#### Now you have the TNW logo on the large screen, which is great, but this logo needs to be removed in the medium screen sizes. This was not informed by the TSE's above because it was not there.
Yours
:point_down:
image

Original
:point_down:
image


### Footer
#### There are still a few things to adjust in the footer. Please do these changes as required by the last TSE's:

- [ ]  Your footer still does not look like the original. This was pointed out to you in a previous review. Please compare your clone with the original page paying close attention to the colors and to the unnecessary whitespace below your footer.

The original page
:point_down:
image

Yours
:point_down:
image


[OPTIONAL]

- I think you could add more padding on the right and the left (on the whole page), just like the original one. 

### Please make the required changes and resubmit for a new code review once you are ready

Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).

**Review by expjazz — CHANGES_REQUESTED** [review]
## [Changes Required]

#### Good work on working on the changes requested, but we still need one more detail:

##### We still have the problem on the footer. If you remove the margin of the last h3 with the class of 'footer__copyright' you will solve these issues. 

### As stated above, the project looks really great! :pray:

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding 

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by bertil291utn** [comment]
# Status: on review 🟠 

##### Do you have any question? *Reach out to me on* [Twitter](bit.ly/bertil-twitter) , [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community as @btandayamo

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  

# Hey guys, you've done a great job

Congratulations guys you've worked so hard on this project I'd like to approve however you still need to make some changes.

First of all 

* [x] Your GitHub flow it looks so professional you mastered it however be careful with your pull request and commits messages must be more meaningful, and also your README file doesn't look like a professional one, next time your project will be rejected. 
  image

image


- [ ] Accomplish according to the actual web page. I'm going to give the exact breakpoints where you have to work and then compare it with yours if it's the same as the original one move to the next breakpoint.

- **425px**
  image



- **768px**

image

- **1024px**

image



- [ ] Certainly, the main goal with this project is to work with responsiveness but you have to work also doing a [clone](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design). So work on fonts, colors, and spaces (paddings and margins), I'm not worried about the content.

  image

Guys please work on these requested changes and request another review, I'm sure the next one you got approved, you don't have reason to rush upon your assignments



> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

Good luck. 
:shipit: ? Not yet

<hr>

##### Do you have any question? *Reach out to me on* [Twitter](bit.ly/bertil-twitter), [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community like @btandayamo

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO @lucianosarno :wave: AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT.

- [x] There still changes required on this project the reason being you haven't yet worked on major issues as pointed out by the previous TSE. 
Most of the sections on your web page as pointed out by @BrittanyBlake don't resemble the original. Kindly you need to go back and check on the suggestions that were given work on them before submitting your project for another review. Your project can not be approved unless this is done.

The same reasons that @peterrobert  and @BrittanyBlake pointed out earlier are the same ones that have prevented me from approving this project. Kindly take a step back and work on them

**If you have any questions feel free to reach out to me via slack** @peterrobert

**Comment by lucianosarno** [comment]
> ### CHANGES REQUIRED ♻️ 🛑
> ### HELLO @lucianosarno 👋 AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT.
> * [ ]  There still changes required on this project the reason being you haven't yet worked on major issues as pointed out by the previous TSE.
>   Most of the sections on your web page as pointed out by @BrittanyBlake don't resemble the original. Kindly you need to go back and check on the suggestions that were given work on them before submitting your project for another review. Your project can not be approved unless this is done.
> 
> The same reasons that @peterrobert and @BrittanyBlake pointed out earlier are the same ones that have prevented me from approving this project. Kindly take a step back and work on them
> 
> **If you have any questions feel free to reach out to me via slack** @peterrobert

Ok Peter, thanks for bringing your considerations and We're sorry for the confusion on this project.
We're going to work on @peterrobert  and @BrittanyBlake suggestions and ask for a new review!

**Comment by bertil291utn** [comment]
# Project Approved ✔️ :shipit:

Hi, guys, your project was approved, please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

[TAKE IN MIND]

- These assignments are also to work on clone skills, next time be careful on fonts, color, and each detail

<hr>

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

##### Do you have any question? *Reach out to me on* [Twitter](bit.ly/bertil-twitter), [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community like @btandayamo

**Pull request merged**